apis/beacon/blocks/blinded_blocks.v2.yaml

```diff
@@ -41,7 +41,7 @@ post:
     - in: header
       schema:
         $ref: '../../../beacon-node-oapi.yaml#/components/schemas/ConsensusVersion'
-      required: false
+      required: true
       name: Eth-Consensus-Version
       description: "Version of the block being submitted, if using SSZ encoding."
```

Comment:
I'm assuming we want this always required for both SSZ & JSON? if so we can probably remove the "if using SSZ encoding" part.

Reply:
yes if it's marked required it's required for all encodings..